runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -552,6 +553,15 @@ public void show()
 			requestFocus();
 			giveClientFocus();
 			log.info("Showing frame {}", frame);
+
+			// Close sidebar if config is set for it.
+			if (config.rememberSidebarState())
+			{
+				if (configManager.getConfiguration(CONFIG_GROUP, CONFIG_CLIENT_SIDEBAR_CLOSED) != null)
+				{
+					toggleSidebar();
```

Comment:
Instead you can just not open the sidebar in the first place by removing the `toggleSidebar` call in `init`.

Reply:
After doing this I realized that it causes a few other problems which I've also resolved, such as saveClientBoundsConfig() defaulting to 809 width to account for an always open sidebar.